bin/concatenate_all.py

```diff
@@ -35,73 +39,83 @@ def main():
         semester = lichen_config_data["semester"]
         course = lichen_config_data["course"]
         gradeable = lichen_config_data["gradeable"]
-        expressions = None
-        if("regex" in lichen_config_data):
-            # this assumes regex is seperated by a ','
-            expressions = lichen_config_data["regex"].split(',')
+
+        # this assumes regex is seperated by a ','
```

Comment:
As Matt suggested in the PR in the main Submitty repo, let's add some error checking here.

These values should be one or more of "submissions", "results", "checkout", right?  Nothing else.

Reply:
Good idea.  Will update.